Gemfile.lock

```diff
@@ -77,4 +77,4 @@ RUBY VERSION
    ruby 3.2.3p157
 
 BUNDLED WITH
-   2.4.19
+   2.6.3
```

Comment:
Given that Apache Spark main repository is still in `2.4.22`, I'm not sure why this is needed. Please revert this.